### ISSUE TITLE
Any chance to see it published on Pypi?

### ISSUE THREAD

**Issue by Guts** [lukasschwab/jsonfeed#7]
Hello @lukasschwab,

Looking both `jsonfeed` and [your profile](https://pypi.org/user/lukass/), I did not find this package published on Pypi.

Any reason to not to?  
Any chance to see it published?

**Comment by lukasschwab** [comment]
No good reason — I'll look into publishing it. One minor issue: PyPI doesn't support namespaces and [`jsonfeed`](https://pypi.org/project/jsonfeed/) is already occupied (by a package that does basically nothing).

For now, I think you can also include this package in requirements.txt files like so: 

```
git+https://github.com/lukasschwab/jsonfeed.git@1.0.0#egg=jsonfeed
```

**Comment by lukasschwab** [comment]
@Guts I've released this package on PyPI as `jsonfeed-util`: https://pypi.org/project/jsonfeed-util/

I also reached out to the owner of the `jsonfeed` package for a name transfer, though I'd be surprised if that request is fruitful: https://github.com/jibaku/jsonfeed/issues/1

**Comment by Guts** [comment]
Thanks you!

> For now, I think you can also include this package in requirements.txt files like so:
> 
> ```
> git+https://github.com/lukasschwab/jsonfeed.git@1.0.0#egg=jsonfeed
> ```

Sure, I was doing this but it does not sounds like a long-term solution to me.

2 additional words:

- your package is built upon feedparser but this one has already a JSON support into its develop branch with an unclear situation avout implementation. See https://github.com/kurtmckee/feedparser/issues/103#issuecomment-1890763789
- your example of parsing in readme (and so on the pypi page) is leading to an AttributeError:

```python
import jsonfeed as jf
import requests

# Requesting a valid JSON feed!
r = requests.get("https://arxiv-feeds.appspot.com/json/test")
# Parse from raw text...
feed_from_text = jf.parse(r.text)
# ...or parse JSON separately.
r_json = r.json()
feed_from_json = jf.Feed.parse(r_json)

```


```python
Traceback (most recent call last):
  File "/home/guts/Git/mkdocs-rss-plugin/tests/dev/dev_json_feed_validation.py", line 7, in <module>
    feed_from_text = jf.parse(r.text)
AttributeError: module 'jsonfeed' has no attribute 'parse'
```

I've no time to investigate more but if I can't make it work, I'll create a specific issue.

Thanks again fro the packaging and publishing work!

**Comment by lukasschwab** [comment]
> your package is built upon feedparser but this one has already a JSON support into its develop branch with an unclear situation avout implementation.

The dependency is only used in a `converters` submodule — it doesn't use `feedparser` for parsing, but allows conversion of a `feedparser`-parsed feed. https://github.com/lukasschwab/jsonfeed/blob/master/jsonfeed/converters/feedparser.py

> your example of parsing in readme (and so on the pypi page) is leading to an AttributeError:

Fixed: 3c2dbd9
